lora/service.go

```diff
@@ -0,0 +1,133 @@
+package lora
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strconv"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/nats-io/go-nats"
+
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/logger"
+)
+
+const protocol = "lora"
+
+// Service specifies an API that must be fullfiled by the domain service
+// implementation, and all of its decorators (e.g. logging & metrics).
+type Service interface {
+	// Create Thing route-map
+	CreateThing(string, string) error
+
+	// Create Thing route-map
+	UpdateThing(string, string) error
+
+	// Remove Thing route-map
+	RemoveThing(string) error
+
+	// Create Channel route-map
+	CreateChannel(string, string) error
+
+	// Remove Channel route-map
+	RemoveChannel(string) error
+
+	// Update Channel route-map
+	UpdateChannel(string, string) error
+
+	// Publish messages on Mainflux NATS broker
+	MessageRouter(Message, *nats.Conn) error
```

Comment:
Please remove NATS connection from this method. Also, rename this method to publish (method name should be a verb). Also, turn on go linter, these comments are in invalid format. Beside that, think about using `mainflux.RawMessage` instead of `Message`.

Reply:
Message is  of type lora.Message here.